Week1/homework/app.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
 {
-  const bookTitles = [
+  const favoriteGames = [
     // Replace with your own book titles
     'harry_potter_chamber_secrets',
+    'call_of_the_wild',
+    'the_turtle',
+    'snowball_and_the_bunny_adventures',
+    'chicken_little',
```

Comment:
I've never played any of those games!
Need to look into it. Cool stuff 👍

Reply:
;) good luck